group_vars/all

```diff
@@ -25,3 +25,5 @@ validate_certs: "yes"
 # env:
 # service_vip:
 # control_interface:
+
+host_capability: "compute, storage, network"
```

Comment:
@vvb 

I still have a minor nit. I think we don't need both `network` and `compute` capabilities. I think we just need two capabilities for now, viz. to say that host provides `disk-storage` and that it `can-run-user-containers`, right? 

I refer the latter as `compute` but I realize it is not a good name. Shall just rename `compute` to something like `can-run-user-containers` if keeps it clear? 

`ceph-mon`, `contiv_storage`, `contiv_network` and `scheduler_stack` needs to run on hosts with this capability.

`ceph-osd` need to run on hosts with `storage` capability.